pkg/api/types.go

```diff
@@ -513,7 +513,8 @@ type EnvVar struct {
 	// Required: This must be a C_IDENTIFIER.
 	Name string `json:"name"`
 	// Optional: no more than one of the following may be specified.
-	// Optional: Defaults to "".
+	// Optional: Defaults to ""; variable references ("$(VAR_NAME)") are
```

Comment:
This is totally open to discussion, but think it's more clear to get rid of the extraneous symbols around the example, leaving something like -
`variable references of the form $(VAR_NAME) are expanded to the value of VAR_NAME according to the variable expansion spec`